src/microbe_stage/MicrobeStage.tscn

```diff
@@ -2091,35 +72,9 @@ __meta__ = {
-[connection signal="OnProcessesPressed" from="MicrobeHUD/BottomLeft/HUDBottomBar" to="MicrobeHUD" method="ProcessPanelButtonPressed"]
-[connection signal="OnStatisticsPressed" from="MicrobeHUD/BottomLeft/HUDBottomBar" to="MicrobeHUD" method="StatisticsButtonPressed"]
-[connection signal="OnSuicidePressed" from="MicrobeHUD/BottomLeft/HUDBottomBar" to="MicrobeHUD" method="OnSuicide"]
-[connection signal="pressed" from="MicrobeHUD/BottomRight/VBoxContainer/Control/Multicellular" to="MicrobeHUD" method="OnBecomeMulticellularPressed"]
-[connection signal="pressed" from="MicrobeHUD/BottomRight/VBoxContainer/Control/Macroscopic" to="MicrobeHUD" method="OnBecomeMacroscopicPressed"]
-[connection signal="mouse_entered" from="MicrobeHUD/BottomRight/EditorButton" to="MicrobeHUD" method="OnEditorButtonMouseEnter"]
-[connection signal="mouse_exited" from="MicrobeHUD/BottomRight/EditorButton" to="MicrobeHUD" method="OnEditorButtonMouseExit"]
-[connection signal="pressed" from="MicrobeHUD/BottomRight/EditorButton" to="MicrobeHUD" method="EditorButtonPressed"]
-[connection signal="hide" from="MicrobeHUD/ProcessPanel" to="MicrobeHUD" method="OnProcessPanelClosed"]
-[connection signal="pressed" from="MicrobeHUD/ScrollContainer/HotBar/Engulfment" to="PlayerMicrobeInput" method="ToggleEngulf"]
-[connection signal="pressed" from="MicrobeHUD/ScrollContainer/HotBar/FireToxin" to="PlayerMicrobeInput" method="EmitToxin"]
-[connection signal="pressed" from="MicrobeHUD/ScrollContainer/HotBar/BindingMode" to="PlayerMicrobeInput" method="ToggleBinding"]
-[connection signal="pressed" from="MicrobeHUD/ScrollContainer/HotBar/UnbindAll" to="PlayerMicrobeInput" method="UnbindAll"]
-[connection signal="pressed" from="MicrobeHUD/ScrollContainer/HotBar/SecreteSlime" to="PlayerMicrobeInput" method="SecreteSlime" binds= [ 1.0 ]]
-[connection signal="Cancelled" from="MicrobeHUD/StagePrototypeConfirm" to="MicrobeHUD" method="OnBecomeMulticellularCancelled"]
-[connection signal="Confirmed" from="MicrobeHUD/StagePrototypeConfirm" to="MicrobeHUD" method="OnBecomeMulticellularConfirmed"]
-[connection signal="OnHelpMenuOpenRequested" from="TutorialGUI" to="MicrobeHUD" method="HelpButtonPressed"]
-[connection signal="OnItemSelected" from="RadialPopup" to="MicrobeHUD" method="OnRadialItemSelected"]
+[connection signal="OnFireToxinButtonPressed" from="MicrobeHUD" to="PlayerMicrobeInput" method="EmitToxin"]
+[connection signal="OnSecreteSlimeButtonPressed" from="MicrobeHUD" to="PlayerMicrobeInput" method="SecreteSlime" binds= [ 1.0 ]]
+[connection signal="OnToggleBindingButtonPressed" from="MicrobeHUD" to="PlayerMicrobeInput" method="ToggleBinding"]
+[connection signal="OnToggleEngulfButtonPressed" from="MicrobeHUD" to="PlayerMicrobeInput" method="ToggleEngulf"]
+[connection signal="OnUnbindAllButtonPressed" from="MicrobeHUD" to="." method="UnbindAll"]
 [connection signal="MakeSave" from="PauseMenu" to="." method="SaveGame"]
-
-[editable path="MicrobeHUD/PausePrompt"]
```

Comment:
It seems the new scene doesn't have this set. Did that lose the pause prompt child scene customizations that were presumably applied in this scene?

Reply:
My Godot shows me the path variable with its value is there.
![image](https://github.com/Revolutionary-Games/Thrive/assets/100433934/f0a58717-e4b6-4275-8004-8537b449d131)

Reply:
You need to right click a node and then click the checbox for "editable child nodes" that shows all the child nodes and then you should be able to replicate any customizations done in the old scene in the newly split scene.

Reply:
This editable thing should be in the microbe HUD scene and not in this scene. This goes against the idea behind trying to contain the GUI stuff to the HUD scene so that this scene is much simplified, still having GUI node changes in this scene for the HUD goes against the basic idea behind the splitting.